worker/include/RTC/SimpleConsumer.hpp

```diff
@@ -87,7 +87,7 @@ namespace RTC
 		RTC::RtpStreamRecv* producerRtpStream{ nullptr };
 		bool keyFrameSupported{ false };
 		bool syncRequired{ false };
-		RTC::SeqManager<uint16_t> rtpSeqManager;
+		std::unique_ptr<RTC::SeqManager<uint16_t>> rtpSeqManager;
```

Comment:
Why is this now a pointer?

Reply:
Because we no longer initialize it in constructor but later within some method and now we must pass an argument (the base offset or whatever).